server/journeys/referral/on-behalf-of/tests.test.ts

```diff
@@ -1,13 +1,24 @@
 import { Express } from 'express'
 import { v4 as uuidv4 } from 'uuid'
 import request from 'supertest'
-import { getByRole } from '@testing-library/dom'
+import { findByText, getByRole, getByText, queryByAttribute } from '@testing-library/dom'
```

Comment:
`import { agent as request } from 'supertest'` can help avoid lint error

Reply:
I'm not having any linting errors here? :)